<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileOfferedPane</class>
 <widget class="QWidget" name="FileOfferedPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>46</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="FileIcon_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Incoming file:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FileName_">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FileSize_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="AcceptButton_">
     <property name="text">
      <string>Accept</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="DeclineButton_">
     <property name="text">
      <string>Decline</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-cancel"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
